mlrun/kfpops.py

```diff
@@ -747,21 +747,34 @@ def format_summary_from_kfp_run(kfp_run, project=None, session=None):
             if error:
                 dag[step]["error"] = error
 
-    short_run = {"graph": dag}
-    short_run["run"] = {
-        k: str(v)
-        for k, v in kfp_run["run"].items()
-        if k
-        in [
-            "id",
-            "name",
-            "status",
-            "error",
-            "created_at",
-            "scheduled_at",
-            "finished_at",
-            "description",
-        ]
+    # align kfp fields to mlrun snake case convention
+    for step in dag:
+        # create snake_case for consistency.
+        # retain the camelCase for compatibility
+        for camelcase_field, snakecase_field in [
+            ("startedAt", "started_at"),
+            ("finishedAt", "finished_at"),
+        ]:
+            if camelcase_field in dag[step]:
+                dag[step][snakecase_field] = dag[step][camelcase_field]
```

Comment:
this can be moved to `mlrun.kfpops.generate_kfp_dag_and_resolve_project`
see:
```
    for node in nodes.values():
        name = node["displayName"]
        record = {
            k: node[k] for k in ["phase", "startedAt", "finishedAt", "type", "id"]
        }